### PR TITLE
Name mapping docs

### PR DESCRIPTION
**Pull request by thoragudf**

### CODE REVIEW

**Comment by alex-tpom6oh** [comment]
Looks great!

File: pages/workspace/tracking-plan/properties.mdx [hunk 1]
```diff
@@ -8,6 +8,18 @@ import Link from '../../../components/Link';
 
 Properties are also known as traits, attributes, metadata, etc. They are information that is attached to events, users or groups that describe the state of the user or the product at the time of the event. Avo allows you to define event, group, user and system properties - each category described below.
 
+## <a name="property-defintion"></a> Property Definition
+
+An property is defined in the event details within the relevant <Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> or in the properties view.
```
Comment (alex-tpom6oh) [inline pages/workspace/tracking-plan/properties.mdx:13]:
```suggestion
A property is defined in the event details within the relevant <Link passHref href="/workspace/tracking-plan/events#a-nameactionsa-actions"><a>actions</a></Link> or in the properties view.
```

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/data-design/name-mapping**](https://docs-91nhlgig9.avo-dev.app/data-design/name-mapping) | [**…e/tracking-plan/events**](https://docs-91nhlgig9.avo-dev.app/workspace/tracking-plan/events) |
|:-:|:-:|
|<a href="https://docs-91nhlgig9.avo-dev.app/data-design/name-mapping">Screenshot of /data-design/name-mapping</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AU3caolx4DjqjMeCEueEyXVMp4c8CBAC9Ckzu7B7Otvcr6YC8T4CrReP6uNraMD29XMCNrwTzzBc-zYwuX-h4A.png">(view full size)</a> |<a href="https://docs-91nhlgig9.avo-dev.app/workspace/tracking-plan/events">Screenshot of /workspace/tracking-plan/events</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/uEGJWXVccJTv8tspCeOhf0Pfyg5e_CikpwjY2oYstQcE9ZjzjcU2M38wJXcrMCTgzItm20n7hUwPmiRsGjH8pQ.png">(view full size)</a> |



| [**…acking-plan/properties**](https://docs-91nhlgig9.avo-dev.app/workspace/tracking-plan/properties) |
|:-:|
|<a href="https://docs-91nhlgig9.avo-dev.app/workspace/tracking-plan/properties">Screenshot of /workspace/tracking-plan/properties</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Zyh-7AW9g7gYnaXqrwP98tz5MPj8vE09C_RI9pmfh19NYA6ffxeTIqFGnhXVMC7UhsZ2g9zGL9mW-XKp0BJ8eQ.png">(view full size)</a> |







Commit 731be67f9fd6701d90b7355cd277de9fa70521e3 (https://docs-91nhlgig9.avo-dev.app).

File: .spelling [hunk 2]
```diff
@@ -266,6 +267,7 @@ setGroup
 set_group
 groupIdentify
 group.set
+wqdasd
```
Comment (alex-tpom6oh) [inline .spelling:270]: oops

**Review by alex-tpom6oh — APPROVED** [review]

**Pull request merged**